### ISSUE TITLE
[BUG] - Translator & CharCounter

### ISSUE THREAD

**Issue by itsxerxes** [mwittrien/BetterDiscordAddons#1797]
**Describe the bug**
Unable to be constructed within Discord once it gets moved to the plugins folder. 

**To Reproduce**
When the plugin gets moved to the plugins folder. 

**Expected behavior**
To be able to enable the plugin and use it. 

**Screenshots**
https://prnt.sc/RLNxkidKTFF3
https://prnt.sc/557FJ8XDwIFc

**Discord Build (stable, ptb or canary):**
Stable.

**Additional context**
Downloaded both plugins from BD's website, once i moved them to the Plugin's folder the above screenshots popped up.

**Comment by JasonHorkles** [comment]
This is caused by the new BD update which broke a lot of plugins

**Comment by itsxerxes** [comment]
Can i do anything to fix it or?

**Comment by cocoflan** [comment]
Yes i have the same problem, all is broken from that BD update, please fix !

**Comment by JasonHorkles** [comment]
> Yes i have the same problem, all is broken from that BD update, please fix !

The dev is actively working on it just be patient

**Comment by ianorlowski** [comment]
![image](https://user-images.githubusercontent.com/37383711/192603759-f03cd0e2-9baf-4684-ad95-fcac8c429e83.png)

**Comment by JasonHorkles** [comment]
It's not all done yet
